aries-site/src/pages/components/tip.mdx

```diff
@@ -44,6 +46,19 @@ Upon hovering, Tip presents additional information about a specific element in a
 
 There are a few different ways that you can use a Tip.
 
+### Truncation Example
+
+Tables are very common to have truncation for many of the content
+since the columns are set at a paticular size. In the example below many of the
+service names are to long so you will need to hover in order to view the full name.
+
+<ExampleImagePreview
+  label="See truncation example in Table"
+  href="/components/table#selecting-multiple-records-and-batch-actions"
+  src="/table-manage-orders-example.png"
```

Comment:
I think most of our other screenshots like this are done in light mode, so for consistency we might want to screenshot this one in light mode as well.

<img width="891" alt="Screen Shot 2021-08-18 at 10 13 56 AM" src="https://user-images.githubusercontent.com/12522275/129943153-af03a7d2-f4a7-429e-85d8-d9a59386f67b.png">

Reply:
okay yes I can do that